ui/shared/status/StatusIconTabButton.qml

```diff
@@ -0,0 +1,57 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtGraphicalEffects 1.13
+import "../../imports"
+import "../../shared"
+import "../../shared/status"
+
+TabButton {
+    id: control
+    width: 40
+    height: 40
+    property color iconColor: Style.current.secondaryText
+    property color disabledColor: iconColor
+    property int iconRotation: 0
+
+    icon.height: 24
+    icon.width: 24
+    icon.color: {
+        if (!enabled) {
+            return control.disabledColor
+        }
+        return (hovered || checked) ? Style.current.blue : control.iconColor
+    }
+
+    onIconChanged: {
+        icon.source = icon.name ? "../../app/img/" + icon.name + ".svg" : ""
+    }
+
+    contentItem: Item {
```

Comment:
First tried to reuse `StatusIconButton` here cause it's doing effectively the same thing. However, putting a `Button` in a `Button` doesn't go well with the built-in mouse events (e.g. clicking the button won't work anymore).

That's why a lot of this had to be replicated from `StatusIconButton`.